R/pkg/R/DataFrame.R

```diff
@@ -2985,8 +2988,18 @@ setMethod("write.df",
             if (is.null(source)) {
               source <- getDefaultSqlSource()
             }
+            cols <- NULL
+            if (!is.null(partitionBy)) {
+              if (!all(sapply(partitionBy, function(c) { is.character(c) }))) {
```

Comment:
not a big deal but `function(c) { is.character(c) }` -> `function(c) is.character(c)` if more changes should be pushed.